config/routes.rb

```diff
@@ -9,6 +9,7 @@
   get '/register' => 'users#new'
   post '/register' => 'users#create'
   get '/dashboard' => 'users#show'
+  patch '/dashboard' => 'users#update'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.